package.json

```diff
@@ -7,7 +7,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "test": "./test/index.sh",
+    "test": "jest && ./test/index.sh",
```

Comment:
this is causing tests to fail

Reply:
It's a different issue (1 unfixed reference after refactoring). Addressed.